scout/static/scout/js/filter.js

```diff
@@ -197,3 +197,9 @@ var Filter = {
         }
     }
 };
+
+/* node.js exports */
+if (typeof exports == "undefined") {
```

Comment:
It's needed, the stuff that uses it just hasn't been committed in the interests of keeping the PR small.